assemblies/assembly_using-ansible-rhdm-in-ONAP/dm/master-docinfo.xml

```diff
@@ -0,0 +1,8 @@
+<productname>{PRODUCT}</productname>
+<productnumber>{PRODUCT_VERSION}</productnumber>
```

Comment:
As discussed, there should be no occurrence of the word "Drools" throughout the doc including the title. Please make the global changes.